app/src/pages/Play/useServerTimeOffset.ts

```diff
@@ -64,25 +65,25 @@ export default function useServerTimeOffset(): number {
       }
 
       // calculate the limit for outliers
-      const roundtrips = samples
-        .filter(successfulSample)
-        .map((sample) => sample.roundtrip)
-      const limit = median(roundtrips) + std(roundtrips)
-
-      const offsets = samples
-        .filter(successfulSample)
-        .filter((sample) => sample.roundtrip < limit) // exclude long request outliers
+      const roundtrips = compact(samples).map((sample) => sample.roundtrip)
+      const limits = [
+        median(roundtrips) - std(roundtrips),
+        median(roundtrips) + std(roundtrips),
+      ]
+
+      // exclude long and short request outliers
+      const offsets = compact(samples)
+        .filter(
+          (sample) =>
+            sample.roundtrip > limits[0] && sample.roundtrip < limits[1]
```

Comment:
Interestingly, I think we can actually leave short requests around. This matches [the algorithm](https://github.com/enmasseio/timesync#algorithm), and I imagine the lower the request latency, the more accurate the offset?